vars/folioDeploy.groovy

```diff
@@ -101,3 +103,51 @@ void greenmail(Project project_config) {
         helm.upgrade("greenmail", project_config.getProjectName(), "''", Constants.FOLIO_HELM_HOSTED_REPO_NAME, "greenmail")
     }
 }
+
+void ldp_server(Project project_config, tenant, admin_user, superadmin_user, db_host, ldp_db_user_password, ldp_queries_gh_token, main_db_password) {
+    String ldpconf = """{
```

Comment:
If we create configMap from this string, may be it would be better to store and template it in the helm chart?

Reply:
Or maybe template engine

Reply:
Reworked with template engine.
@OHaimanov, @vitalii-ustalu Please check now.